### PR TITLE
Adds Large Welding Fuel Tanks

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Adds a large fuel tank, it's like the normal fuel tank but bigger, what else did you expect.
They can be found either round start in engineering, or at cargo. 
Tanks hold 5000 units of welding fuel, I hope you weren't expecting a 100000 unit fuel tactical nuke.

## Obligatory Image
unknown

## Why It's Good For The Game
So you know how like, botany and atmos get those giant water tanks with huge fuckoff amounts of water in them that nobody uses? I thought that we might as well give engineers something like that, for the super rare off chance that someone actually needs 5,000 units of welding fuel.

## Changelog
:cl:
add: Large fuel tank, able to be ordered from cargo if you're into that sorta thing
tweak: Replaced some fuel tanks in engineering with their larger variants
/:cl:

### CODE REVIEW

**Comment by GunHog** [comment]
Thank you for enabling more ways to blow stuff up 👍

**Comment by ATH1909** [comment]
When these go boom, how big is the boom?

**Comment by Dorsisdwarf** [comment]
> 
> When these go boom, how big is the boom?
@ATH1909 

### **For your convenience:**
These proposed highcaps use `explosion(get_turf(src), 1, 2, 7, flame_range = 12)`

As opposed to the standard tanks, which use `explosion(get_turf(src), 0, 1, 5, flame_range = 5)`

### **Demonstration**
**Demonstration A**
Before standard tank blast: 
ATH1909 is lazy
After standard tank blast:
Lazy boy

**Demonstration B**
Before High Capacity tank blast:
Read the code before asking this kind of question
After High Capacity tank blast:
It's very simply laid out for your convenience

### Conclusion

When the high capacity fuel tanks go boom, the boom is bigger than the boom when the normal fuel tanks go boom. Thank you for reading.

**Comment by Pugie** [comment]
So welding fuel tanks that can gib people in layman's terms. Fun.

**Comment by Pugie** [comment]
This is literally just being added so it's an abusable gib it serves no practical purpose, interesting how you didn't mention this paxil

**Comment by EdgeLordExe** [comment]
we already have much less destructive, click the person to kill them tools, like elances. And elances dont kill you if you do them correctly, this is essentially a much much much shittier version of them

**Comment by Pugie** [comment]
That doesn't change the fact powercreeps are garbage.

**Review by Twaticus — APPROVED** [review]

**Comment by Fikou** [comment]
> This is literally just being added so it's an abusable gib it serves no practical purpose, interesting how you didn't mention this paxil

just dont get hit

**Comment by Krysonism** [comment]
Welderbombs are neat, maybe remove the gib if that is a concern.

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-1). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.
